frontend/Dockerfile.dev

```diff
@@ -0,0 +1,29 @@
+FROM node:20-alpine
+
+WORKDIR /frontend
+
+# Install dependencies based on the preferred package manager
+COPY package.json yarn.lock* package-lock.json* pnpm-lock.yaml* ./
+RUN \
+  if [ -f yarn.lock ]; then yarn --frozen-lockfile; \
+  elif [ -f package-lock.json ]; then npm ci; \
+  elif [ -f pnpm-lock.yaml ]; then yarn global add pnpm && pnpm i; \
+  # Allow install without lockfile, so example works even without Node.js installed locally
+  else echo "Warning: Lockfile not found. It is recommended to commit lockfiles to version control." && yarn install; \
+  fi
+
+COPY . .
+
+# Next.js collects completely anonymous telemetry data about general usage. Learn more here: https://nextjs.org/telemetry
+# Uncomment the following line to disable telemetry at run time
+# ENV NEXT_TELEMETRY_DISABLED 1
+
+# Note: Don't expose ports here, Compose will handle that for us
+
```

Comment:
Maybe remove these unused comments, I don't think we would need these in the future.

Reply:
removed [c0c9d52](https://github.com/KernelKrakens/FinBuddy/pull/5/commits/c0c9d52e239cba7f6f02339f4c80bd0a03486d86)